src/status_im/contexts/wallet/wallet_connect/events.cljs

```diff
@@ -204,10 +198,59 @@
                                          {:version version}))}]]]}
        {:fx [[:dispatch [:wallet-connect/pair scanned-text]]]}))))
 
+;; We first load sessions from database, then we initiate a call to Wallet Connect SDK and
+;; then replace the list we have stored in the database with the one that came from the SDK.
+;; In addition to that, we also update the backend state by marking sessions that are not
+;; active anymore by calling `:wallet-connect/disconnect-session`.
+(rf/reg-event-fx
+ :wallet-connect/fetch-active-sessions-success
+ (fn [{:keys [db]} [sessions]]
+   (let [time-now           (-> (time/now)
+                                time.coerce/to-long
+                                (/ 1000))
```

Comment:
Suggestion: The `now` cofx in the event serves the timestamp and we can utilize it here.